tests/foreman/cli/test_repository.py

```diff
@@ -945,29 +945,30 @@ def test_positive_synchronize_docker_repo_with_tags_whitelist(self, repo_options
                     'content-type': 'docker',
                     'docker-upstream-name': CONTAINER_UPSTREAM_NAME,
                     'url': CONTAINER_REGISTRY_HUB,
-                    'mirror-on-sync': 'no',
+                    'mirroring-policy': 'additive',
                 }
             ]
         ),
         indirect=True,
     )
-    def test_positive_synchronize_docker_repo_set_tags_later(self, repo):
+    def test_positive_synchronize_docker_repo_set_tags_later_additive(self, repo):
         """Verify that adding tags whitelist and re-syncing after
         synchronizing full repository doesn't remove content that was
-        already pulled in
+        already pulled in when mirroring policy is set to additive
 
         :id: 97f2087f-6041-4242-8b7c-be53c68f46ff
 
         :parametrized: yes
 
-        :expectedresults: Non-whitelisted tags are not removed
+        :expectedresults: Non-whitelisted tags are not removed when
+        mirroring policy is set to additive
```

Comment:
This is causing testimony validation to fail because it doesn't handle the newline without an indent or without a numbered list. 

I don't think its really providing much anyway, as the description of the test already includes this expectation.

I'd probably just remove it.